Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Mesh_3_plugin.cpp

```diff
@@ -75,6 +75,7 @@ class Mesh_3_plugin :
              WRITE set_sharp_edges_angle_bound)
   Q_PROPERTY(double edges_sizing READ get_edges_sizing WRITE set_edges_sizing)
   Q_PROPERTY(double edges_min_sizing READ get_edges_min_sizing WRITE set_edges_min_sizing)
+  Q_PROPERTY(double edges_distancing READ get_edges_distancing WRITE set_edges_distancing)
```

Comment:
"distancing" is not really what we mean. For consistency with `approx` below (which stands for `facet_distance`), it could be renamed `edges_approx`